pkg/orchestrator/config/config.go

```diff
@@ -51,10 +52,11 @@ func NewDefaultOrchestratorConfig() *OrchestratorConfig {
 		panic(err)
 	}
 	oc := OrchestratorConfig{
-		Scrubber:              redact.NewDefaultDataScrubber(),
-		MaxPerMessage:         100,
-		OrchestratorEndpoints: []apicfg.Endpoint{{Endpoint: orchestratorEndpoint}},
-		PodQueueBytes:         15 * 1000 * 1000,
+		Scrubber:                 redact.NewDefaultDataScrubber(),
+		MaxPerMessage:            100,
+		MaxWeightPerMessageBytes: 5000000,
```

Comment:
We chunk messages by size before being compressed. But the 5MiB limit of intake is on messages already compressed. What do you think the best threshold `MaxWeightPerMessageBytes` here ?

Reply:
Our compression as of right now is really strong if you look at our internal graphs (compressed vs decompressed). You could look a bit into our data/graphs to see those comparisons and make a decision based on that